bin/node/src/commands/bundled.rs

```diff
@@ -175,12 +185,21 @@ impl BundledCommand {
             (rpc_id, "rpc"),
         ]);
 
+        // Start network transaction builder. The endpoint is available after loading completes.
+        // SAFETY: socket addr yields valid URLs
+        let store_ntx_builder_url = Url::parse(&format!(
+            "http://{}:{}/",
+            store_ntx_builder_address.ip(),
+            store_ntx_builder_address.port()
+        ))
```

Comment:
I think `SocketAddr::to_string()` actually does the right thing here as well 

```suggestion
        let store_ntx_builder_url = Url::parse(&store_ntx_builder_address.to_string());
```